src/components/CreateCoursePage/__snapshots__/CreateCoursePage.test.jsx.snap

```diff
@@ -162,7 +162,6 @@ exports[`CreateCoursePage renders page correctly with course create error 1`] =
               "timeout": 300,
               "unmountOnExit": false,
             },
-            "displayName": "Fade",
```

Comment:
any ideas why this attr has been removed from various snapsots?

Reply:
Snapshots are updating when we update the `@edx/paragon` version from `20.39.3` to [20.40.0](https://github.com/openedx/paragon/compare/v20.39.3...v20.40.0).